src/actions/ExpandableContent.tsx

```diff
@@ -1,21 +1,29 @@
 import React, { useState } from "react"
 
+type Order = "above" | "below"
```

Comment:
This could be an enum as well. But since it's only 2 option, I'm ok with this as well!